demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -336,6 +336,12 @@ def is_valid_default_argument_in_reputation_command(self):
                             if self.handle_error(error_message, error_code, file_path=self.file_path):
                                 self.is_valid = False
                                 flag = False
+                        if arg.get('isArray') is False:
```

Comment:
Seems like a boolean field, what other falsy value could it have?
```suggestion
                        if not arg.get('isArray'):
```